assets/js/base/components/cart-checkout/cart-line-items-table/cart-line-item-row.tsx

```diff
@@ -237,13 +237,16 @@ const CartLineItemRow: React.ForwardRefExoticComponent<
 				</td>
 				<td className="wc-block-cart-item__product">
 					<div className="wc-block-cart-item__wrap">
-						<ProductName
-							disabled={
-								isPendingDelete || isProductHiddenFromCatalog
-							}
-							name={ name }
-							permalink={ permalink }
-						/>
+						<span className="wc-block-components-product-title">
```

Comment:
I believe wrapping this with a classname is the wrong approach. It looks as though `wc-block-components-product-title` is used within the product title block and is not exposed as a component. **Why don't we expose a component instead?** if the styles and component are to be shared, that makes the most sense.

- Create a base component called product title
- Move the code from the product title block into the component. Note that it accepts `tag` so you can define the h* tag or pass a span.
- Make both the product title block and the cart item list use the new component.

Reply:
> I believe wrapping this with a classname is the wrong approach. It looks as though `wc-block-components-product-title` is used within the product title block and is not exposed as a component. **Why don't we expose a component instead?** if the styles and component are to be shared, that makes the most sense.
> 
> * Create a base component called product title
> * Move the code from the product title block into the component. Note that it accepts `tag` so you can define the h* tag or pass a span.
> * Make both the product title block and the cart item list use the new component.

Thanks for your review, @mikejolley. Creating the base component sounds like a good approach. I'll do that now.